packages/world/src/interfaces/IWorldKernel.sol

```diff
@@ -101,5 +101,5 @@ interface IWorldCall {
  * registered functions selectors from the `CoreModule`.
  */
 interface IWorldKernel is IWorldData, IWorldModuleInstallation, IWorldCall, IErrors {
-
+  event HelloWorld();
```

Comment:
wasn't sure where to put this cc @alvrs @dk1a

Reply:
also just wanted to note that I considered event names like `Initialized` and `Constructed` and `Deployed`, but I thought `HelloWorld` was a nice callback to beginner programming tutorials and fits nicely into our "world" terminology :)

Reply:
Fine to put it on `IWorldKernel`. I like `HelloWorld`!